theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -466,6 +466,19 @@ def test_pool_grad(self):
             )
 
 
+# this has been a problem in the past
+def test_dnn_conv_border_mode():
```

Comment:
I did not check if it raises the error before the fix, no.  But I just did.

Also the error comes from flake8, not from this test (which is on GPU).